lib/packages/docutils/rstgen.nim

```diff
@@ -1355,6 +1355,14 @@ proc rstToLatex*(rstSource: string; options: RstParseOptions): string {.inline,
   rstGenera.renderRstToOut(rstParse(rstSource, "", 1, 1, option, options), result)
 
 
+proc rstToJson*(rstSource: string; options: RstParseOptions): string {.inline, since: (1, 3).} =
```

Comment:
* why not use the hook mechanism in jsonutils?
* badly named since it returns string, not `JsonNode`; `rstToJsonString` would be more accurate
* this doesn't compose; instead, the hook mechanism in `jsonutils` would allow composition, eg:
```nim
type RstSource = distinct string
let b = ("foo", @[RstSource"*hello*", RstSource"**world**"]).toJson
```